src/main/java/org/folio/dcb/service/impl/CustomTenantService.java

```diff
@@ -0,0 +1,41 @@
+package org.folio.dcb.service.impl;
+
+import lombok.extern.log4j.Log4j2;
+import org.folio.dcb.config.props.TestTenant;
+import org.folio.spring.FolioExecutionContext;
+import org.folio.spring.liquibase.FolioSpringLiquibase;
+import org.folio.spring.service.PrepareSystemUserService;
+import org.folio.spring.service.TenantService;
+import org.folio.tenant.domain.dto.TenantAttributes;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.context.annotation.Primary;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Service;
+
+@Log4j2
+@Service
+@Primary
+@Lazy
+public class CustomTenantService extends TenantService {
+
+  private final PrepareSystemUserService systemUserService;
+  private final TestTenant testTenant;
+
+
+  public CustomTenantService(JdbcTemplate jdbcTemplate, FolioExecutionContext context,
+                             FolioSpringLiquibase folioSpringLiquibase, PrepareSystemUserService systemUserService, TestTenant testTenant) {
+    super(jdbcTemplate, context, folioSpringLiquibase);
+
+    this.systemUserService = systemUserService;
+    this.testTenant = testTenant;
+  }
+
+  @Override
+  protected void afterTenantUpdate(TenantAttributes tenantAttributes) {
+    log.debug("afterTenantUpdate:: parameters tenantAttributes: {}", tenantAttributes);
+    if (!context.getTenantId().startsWith(testTenant.getTenantName())) {
```

Comment:
I think, this check is not needed here. @gurleenkaurbp @steveellis @kalibek-epam , Do we need to have the check similar to Mod-Inn-Reach.

Reply:
@Vignesh-kalyanasundaram, @gurleenkaurbp , @kalibek-epam, @steveellis 
FYI
for the mod-inn-reach this check was introduced only for testing purposes as well as TestTenant class.
Just to avoid triggering system-user setup mechanism via system-user service during the JUnit tests or local test env DB setup.

So I removed such functionality for now.